tests/test_run_service.py

```
@@ -427,7 +427,7 @@ def test_list_runs_paginated_returns_total_and_slice(self, tmp_path, monkeypatch
             storage.write_summary(run_id, summary, runs_dir=tmp_path)
 
         # Mock storage functions to use only our test runs
-        monkeypatch.setattr(storage, "list_runs", lambda runs_dir=None: run_ids)
+        monkeypatch.setattr(storage, "list_runs", lambda runs_dir=None: sorted(run_ids))
```

Ruff (0.14.14)

Unused lambda argument: `runs_dir`

(ARG005)